joern-cli/frontends/csharpsrc2cpg/src/test/scala/io/joern/csharpsrc2cpg/passes/DependencyTests.scala

```diff
@@ -165,4 +165,30 @@ class DependencyTests extends CSharpCode2CpgFixture {
       }
     }
   }
+
+  "a csproj file specifying a dependency with whitespaces in between" should {
+    val config = Config().withDownloadDependencies(true);
+    "not throw a exception" in {
+      val cpg = code("""
+          |namespace Foo;
+          |""".stripMargin)
+        .moreCode(
+          """
+            |<Project Sdk="Microsoft.NET.Sdk">
+            |    <ItemGroup>
+            |      <PackageReference Include=" System.Security.Cryptography.Pkcs" Version="6 .0.4"/>
+            |      <PackageReference Include="CommandLineParser" Version="2.9.1" />
+            |    </ItemGroup>
+            |</Project>
+            |""".stripMargin,
+          "Foo.csproj"
+        )
+        .withConfig(config)
+
+      inside(cpg.dependency.l) { case dep :: dep2 :: Nil =>
+        dep.name shouldBe " System.Security.Cryptography.Pkcs"
```

Comment:
@karan-batavia Shouldn't we trim the white space?

Reply:
This hasn't changed from before.

Reply:
That is ok, but in general shouldn't we store trimmed values. Does it break anything
cc - @tuxology @pandurangpatil